db/schema.rb

```diff
@@ -19,6 +19,8 @@
     t.string "message"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "user_id"
```

Comment:
what is t.bigint???

Reply:
It is a Postgres data type - allows for really big integers (assuming we gain millions of Acebook users!)
It allows numbers btwn **-9,223,372,036,854,775,808** &  **+9,223,372,036,854,775,807**